config/base.js

```diff
@@ -39,6 +39,12 @@ const SCHEMA_CHILD_PIPELINES = Joi.object()
         startAll: Joi.boolean()
     })
     .unknown(false);
+
+const SCHEMA_SUBSCRIBE = Joi.object()
+    .keys({
+        scmUrls: Joi.array().items(Joi.object())
```

Comment:
```suggestion
        scmUrls: SCHEMA_SCM_URLS.required()
```

could reuse predefined scm urls

Reply:
or if the regex is not the same, might be better to use a different key

Reply:
Yeah, I should probably go with the different key.